environments/in-app-messaging/in-app-messaging-with-pinpoint-campaign/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "name": "in-app-messaging-with-pinpoint-campaign-environment",
+  "version": "0.0.1",
+  "scripts": {
+    "pull": "amplify pull --appId d19pc24nlvj2b3 --envName staging"
```

Comment:
Is this environment in us-east-2?

Reply:
oh no
it is in us-west-2
let me update the env files

Reply:
Updated env to be us-east-2